clang/lib/Driver/ToolChains/Linux.cpp

```diff
@@ -256,6 +256,14 @@ Linux::Linux(const Driver &D, const llvm::Triple &Triple, const ArgList &Args)
       ExtraOpts.push_back("-z");
       ExtraOpts.push_back("max-page-size=16384");
     }
+    if (Triple.isAndroidVersionLT(29)) {
```

Comment:
this says "<" but the commit message says "<="?

Reply:
Edited the commit message

Reply:
i meant "Add no-rosegment by default for API 29 and earlier." --- i actually hadn't noticed the _pull request_ message. i think that was already right? (or at least, "consistent with the code and code comments".)